pkgs/development/libraries/openmpi/default.nix

```diff
@@ -193,7 +193,7 @@ stdenv.mkDerivation (finalAttrs: {
         ];
         part2 = builtins.attrNames wrapperDataSubstitutions;
       };
-    in
+    in lib.optionalString stdenv.isLinux
```

Comment:
I'd prefer to only condition the substitution and not all of the postInstall... We might enable multiple outputs in the future for other platforms, and you'd still want to delete all `*.la` files...

Reply:
Then we need extra condition for the split output moves too. If we want to enable multiple outputs in the future for other platforms we can make it more granular later? I do not know what darwin does with .la files. I know that they are not needed on Linux.

Reply:
> Then we need extra condition for the split output moves too.

No you won't. That was the purpose of 2a4636b06cee .

> If we want to enable multiple outputs in the future for other platforms we can make it more granular later?

The diff will be much smaller thanks to 2a4636b06cee .

> I do not know what darwin does with .la files. I know that they are not needed on Linux.

Yea that line was peculiar to me so I didn't touch it.. I suspect they might be generated on Darwin as well, so I wouldn't touch it even now. For sure it doesn't hurt if these files are not found at all.